utils/utils.go

```diff
@@ -130,6 +130,8 @@ func GetTemplateFuncs() template.FuncMap {
 		"formatAddressAsLink":                     FormatAddressAsLink,
 		"formatBuilder":                           FormatBuilder,
 		"formatDifficulty":                        FormatDifficulty,
+		"getAvailableCurrencies":                  price.GetAvailableCurrencies,
```

Comment:
Any reason why the available currencies are passed as a template function instead of an extra field in PageData?

Reply:
I had one ... but after thinking about - you are right ... put it in the PageData now